### DISCUSSION TITLE
WSO Number

### DISCUSSION THREAD

**Discussion by eflashlogistics** [code4recovery/12-step-meeting-list#747] [Q&A] [answered]
Hi! What would be the best practice for adding our WSO number? We post it now in our old listing format. I added a custom field, but I would need to modify the import template, or add these manually on wordpress correct? We are a small district so that is easy.

**Comment by brianw-area24** [comment, marked as answer]
A couple of options for you:

- Include it with the group name (Name of Group - GSO number)
- Include it in the Group Notes field

I'm not sure how you added a custom field, but I'm thinking you might have to make changes each time we update the plugin, and that's probably not something you want to do.
